games-engines/openscenegraph/openscenegraph-3.6.3.recipe

```diff
@@ -10,7 +10,7 @@ virtual reality industries."
 HOMEPAGE="http://www.openscenegraph.org/"
 COPYRIGHT="2002-2018 Robert Osfield and others"
 LICENSE="OSGPL"
-REVISION="1"
+REVISION="2"
```

Comment:
@kenmays, you should probably update your master branch and rebase your PR against 3e709df1958efa83eecc90bfe5044b4baff709e3 or any newer commit, and then:
```diff
-REVISION="2"
+REVISION="3"
```

Also, I think it might be nice to revert the minor cleanups, i.e. keep the dependencies which are commented out, if @threedeyes added them on purpose. I guess there might be plans to uncomment these dependencies in the future.